arches_controlled_lists/src/arches_controlled_lists/components/tree/AddDeleteControls.vue

```diff
@@ -38,7 +38,7 @@ import type {
     IconLabels,
     RowSetter,
     Selectable,
-} from "@/arches_references/types";
+} from "../../types";
```

Comment:
Was this intentional (throughout)?

Reply:
not intentional, must have accidentally let vscode update the paths. taken care of in latest